packages/kusama/src/__snapshots__/assetHubKusama.scheduler.e2e.test.ts.snap

```diff
@@ -94,6 +94,68 @@ exports[`Asset Hub Kusama Scheduler E2E tests > cancelling a scheduled task is p
 ]
 `;
 
+exports[`Asset Hub Kusama Scheduler E2E tests > execution of scheduled preimage lookup call works > events for scheduled lookup-task execution 1`] = `
+[
+  {
+    "data": {
+      "new_": "(rounded 16000000000000000000)",
+      "old": "(rounded 16000000000000000000)",
+    },
+    "method": "TotalIssuanceForced",
+    "section": "balances",
+  },
+  {
+    "data": {
+      "id": null,
+      "result": "Ok",
+      "task": [
+        "(rounded 27000000)",
```

Comment:
this number should be redacted
https://github.com/open-web3-stack/polkadot-ecosystem-tests/actions/runs/13886161376/job/38851168311

Reply:
Will fix.